internal/v1/api.yaml

```diff
@@ -1294,6 +1315,21 @@ components:
           type: string
         signature:
           type: string
+    UploadPackageRequest:
+      type: object
+      properties:
+        packages:
+          type: array
+          items:
+            type: string
+        recommendedPackages:
+          type: integer
+          format: int32
+    RecommendationsResponse:
+      type: object
+      properties:
+          message:
+            type: string
```

Comment:
The original API returns an array in string weirdly: `"[package1, package2]"`

I asked the plan is to break it up into a proper JSON string, therefore, I suggest to expose it here correctly as a JSON array and for now parse the string and break it up manually until this is fixed upstream.